spec/integration/update_subscription_list_spec.rb

```diff
@@ -0,0 +1,59 @@
+RSpec.describe "Updating a subscriber list", type: :request do
+  let!(:subscriber_list) { create(:subscriber_list) }
+  let(:update_subscriber_lists_path) { "/subscriber-lists/#{slug}" }
+  let(:slug) { subscriber_list.slug }
+  let(:update_params) do
+    {
+      "title" => "A new title",
+    }
+  end
+
+  context "with authentication and authorisation" do
+    before do
+      login_with_internal_app
+    end
+
+    it "returns a 200" do
+      patch update_subscriber_lists_path, params: update_params.to_json, headers: json_headers
+      expect(response.status).to eq(200)
+    end
+
+    it "returns a the subscription with updated title" do
+      patch update_subscriber_lists_path, params: update_params.to_json, headers: json_headers
+
+      expect(JSON.parse(response.body).dig("subscriber_list", "title")).to eq(update_params["title"])
+    end
+
+    it "returns 422 if no allowed updatable parameters are provided" do
```

Comment:
If you do want to forbid nulls, you should have a test for `params: { title: nil }.to_json`